build/push-ci-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pushes a continuous integration build to our official CI repository
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+LATEST=$(git describe)
```

Comment:
I like using `git describe` but there is a problem -- it won't sort lexicographically.  It is of the form `v0.7.0-348-g58abb40`.  The version number will work but the number of patches since the tag isn't zero padded.  We'll also face this issue when we hit `v0.10`.

Perhaps mix the date-time in there?

Or we can just declare it not a problem and do the sorting client side when we present this stuff.